taca_ngi_pipeline/cli.py

```diff
@@ -74,6 +75,10 @@ def deliver(ctx, deliverypath, stagingpath,
             default=None,
             type=click.STRING,
             help='pi-email, to be specified if PI-email stored in statusdb does not correspond SUPR PI-email')
+@click.option('--pi-name',
```

Comment:
This will soon be changed to PI email, so we can keep using the existing option for it 
https://github.com/ScilifelabDataCentre/dds_web/pull/1084